src/ctia/http/routes/common.clj

```diff
@@ -99,6 +108,28 @@
     {:gte from
      :lt to-or-now}))
 
+(s/defn prep-es-fields-schema :- (s/protocol s/Schema)
+  "Conjoins ES :fields onto search-parameters."
+  [search-query-params :- (s/maybe (s/protocol s/Schema))]
+  (let [fields-schema  (st/get-in search-query-params [:fields])
+        default-fields (-> fields-schema first :vs
+                           (set/difference (set (keys em/base-entity-mapping)))
```

Comment:
We cannot rely on the store layer in a route layer. Moreover this is specific to the es store. There is a schema for these base fields.

Reply:
Ah, right. Good point. 👍